agml/synthetic/synthetic_data_generation/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set( PLUGINS "lidar;visualizer;canopygenerator;syntheticannotation" )
 cmake_minimum_required(VERSION 2.4)
 project(helios)
 SET(CMAKE_CXX_COMPILER_ID "GNU")
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -stdlib=libc++ -std=c++17")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -std=c++17")
```

Comment:
The flag `-stdlib=libc++` does not work for some compilers and causes the build to fail. It also shouldn't be necessary.